tests/include

```diff
@@ -0,0 +1 @@
+../include
```

Comment:
Is there a reason why we would want this instead of adjusting the cmake include dir?